Makefile

```diff
@@ -1,34 +1,15 @@
 VERSION := $(shell python -c "exec(open('''tower_cli/constants.py''').read(), globals()); print VERSION")
+RELEASE := $(shell python -c "exec(open('''tower_cli/constants.py''').read(), globals()); print RELEASE")
 
-DISTS = el6 el7
-DIST_SUFFIX_el6 = 
-DIST_SUFFIX_el7 = .centos
-MOCK_CFG_el6 = epel-6-x86_64
-MOCK_CFG_el7 = epel-7-x86_64
+el6: dist/ansible-tower-cli-$(VERSION).tar.gz rpm-build/ansible-tower-cli-${VERSION}.spec
+	mock -r epel-6-x86_64 --buildsrpm --spec rpm-build/ansible-tower-cli-${VERSION}.spec --sources dist/ --resultdir rpm-build
+	mock -r epel-6-x86_64 --rebuild rpm-build/ansible-tower-cli-3.3.0-1.el6.src.rpm --resultdir rpm-build
 
-.PHONY: all clean $(DISTS)
-.DEFAULT_GOAL = all
+el7: dist/ansible-tower-cli-$(VERSION).tar.gz rpm-build/ansible-tower-cli-${VERSION}.spec
+	mock -r epel-7-x86_64 --buildsrpm --spec rpm-build/ansible-tower-cli-${VERSION}.spec --sources dist/ --resultdir rpm-build
+	mock -r epel-7-x86_64 --rebuild rpm-build/ansible-tower-cli-3.3.0-1.el7.src.rpm --resultdir rpm-build
```

Comment:
Can you update this to be something like `rpm-build/ansible-tower-cli-$(VERSION)-$(RELEASE)`?

Reply:
For clarification are you looking for the make target to be rpm-build/ansible-tower-cli-$(VERSION)-$(RELEASE) or the tar.gz/spec file or something else?